src/crimson/net/SocketMessenger.h

```diff
@@ -37,13 +37,17 @@ class SocketMessenger final : public Messenger {
   using Throttle = ceph::thread::Throttle;
   ceph::net::PolicySet<Throttle> policy_set;
   const std::string logic_name;
+  const uint64_t nonce;
```

Comment:
could you use `uint32_t` to be consistent with `entity_addr_t::nonce` ? also, please note, in glibc, `pid_t` is represented using `int`.

Reply:
fixed